stardis/transport/base.py

```diff
@@ -62,7 +64,14 @@ def calc_weights(delta_tau):
     return w0, w1
 
 
-def single_theta_trace(stellar_model, alphas, tracing_nus, theta):
+@numba.njit()
+def single_theta_trace(
+    stellar_model_fv_geometry_cell_length,
+    stellar_model_boundary_temp,
```

Comment:
[Opinionated] you can drop `stellar_model_` it's making names too long 

Also, it's `boundary_temps` (plural)